app/commands/help.js

```diff
@@ -12,7 +12,13 @@ module.exports = Base => class extends Base {
         this.helpFullDescription = {};
     }
 
-    welcomeMessage(playerName) {
+    async welcomeMessage(playerName) {
```

Comment:
I don't really like adding to this to the login message.. maybe adding a command like `!xp autostore status`

Reply:
I can do that, i like informing players the status of their autostore when they log in though, it seems like very useful information you'd want to have first thing when you get in. i've separated it into it's own function and call it in the handlePlayerLogin method.

Reply:
I think I'll also add ```!xp check autostore``` because i like having check be the thing you use if you wanna check on something xp related.